tools/mtmd/clip.cpp

```diff
@@ -1560,6 +1606,27 @@ struct clip_graph {
         }
     }
 
+    // siglip2 naflex
+    ggml_tensor * resize_position_embeddings() {
```

Comment:
This is cool! It should be reuse-able for https://github.com/ggml-org/llama.cpp/issues/14318 :)